.github/workflows/ci.yaml

```diff
@@ -185,9 +185,70 @@ jobs:
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
       - name: Build and push
         id: docker_build
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
+        with:
+          file: dse-68/Dockerfile.${{ matrix.jdk-version }}
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          platforms: linux/amd64
+          target: dse68
+
+  publish-dse-ubi:
+    name: Publish DSE UBI image
+    if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
+    needs: build-dse
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        jdk-version: [ubi7]
```

Comment:
Wrong variable name I reckon. Also can we have JDK 11 and 9 versions of UBI7 or no?

Reply:
Yeah, the variable name probable should be changed here.
However, the official DSE images only use JDK8 for the UBI images, so I'm gonna go with "no" we can't have JDK 11 or 9 versions of UBI 7.